pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -41,6 +42,7 @@
 , stripBytecode ? true
 , includeSiteCustomize ? true
 , static ? stdenv.hostPlatform.isStatic
+, enableFramework ? false
```

Comment:
Name shouldn't be `frameworks` because it is a name and not a plural?

Reply:
The configure flag being `--enable-framework`, I guess `enableFramework` is more direct.